enhancement: Logo on Navbar implemented

ckhorra1 opened the pull request:
Wyatt ,Ari, and  Cyrus both imported the icon logo and made it as button. Once the button is clicked it will take the user to the landing page. The navbar will follow to any page it goes to as well as it will still stay at the top while the user scrolls down a page. issue #15

ckhorra1 committed:
feat: Logo on Navbar implemented

Wyatt and Cyrus both imported the icon logo and made it as button. Once the button is clicked it will take the user to the landing page. The navbar will follow to any page it goes to as well as it will still stay at the top while the user scrolls down a page.

zxkevin reviewed (changes requested):
Though not in AC, please add onClick for the Create Post button to just navigate to /CreatePost even if the page is empty

Nitpick: change the index.html file so that our page title isn't React App anymore

ckhorra1 committed:
enhancement: Updated title and mock page

Updated the title to Warai and made it so there is a mock Create Post page when the user clicks on the Post button on the navbar.

zxkevin commented on packages/hackmcx-client/src/CreatePost.js:
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
Don't need this file. Please remove

zxkevin commented on packages/hackmcx-client/src/App.js:
```diff
@@ -8,6 +9,7 @@ function App() {
 			<Router>
 				<Routes>
 					<Route exact path='/' exact element={<LandingPage />} />
+					<Route exact path='/CreatePost' exact element={<CreatePost />} />
```
Didn't need this, but why not

zxkevin commented on packages/hackmcx-client/src/LandingPage.js:
```diff
@@ -40,7 +40,13 @@ const LandingPage = () => {
+							<Button
+								variant='contained'
+								onClick={() => {
+									navigate('/CreatePost');
```
This looks good!

zxkevin reviewed (changes requested):
Check comments on code

ckhorra1 committed:
Updated CreatePost.js

ckhorra1 committed:
Update app.js and removed creatpost.js

I removed a mock createpost.js file that I created when it was not needed. I also revised app.js where in the routes tag I had <Route exact path='/CreatePost' exact element={<CreatePost />} />. Removed it.

zxkevin reviewed (approved):
This looks good!

ckhorra1 commented:
Thanks for  the reviews!
